Plugins/NativeEngine/Include/Babylon/ShaderCache.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include <string_view>
+#include <memory>
+
+namespace Babylon
+{
+    namespace ShaderCache
+    {
+        void Enable(bool enabled);
+        bool IsEnabled();
```

Comment:
I think for these types of "property" like members, elsewhere we have done more like:
```suggestion
        void Enabled(bool enabled);
        bool Enabled();
```
I can't seem to find any examples, maybe @bghgary can comment.

Reply:
Fixed

Reply:
Here is one: https://github.com/BabylonJS/UrlLib/blob/main/Include/UrlLib/UrlLib.h#L48-L50